docs/en/cli.md

```diff
@@ -0,0 +1,475 @@
+# OpenMLDB operation CLI
+
+* [ns client](#ns-client)
+* [tablet client](#tablet-client)
+
+### NS Client
```

Comment:
## NS Client
(the Chinese doc also has this incorrect heading level issues, please correct this for the English doc)

Reply:
Ok, thank you.